### PR TITLE
Create IntakeRollerSubsystem and add Commands

### PR DESCRIPTION
**Pull request by Tyson453**

### CODE REVIEW

File: cowtown2023/src/main/java/com/team2357/frc2023/commands/intakeRoller/IntakeRollerEjectCubeCommand.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.team2357.frc2023.commands.intakeRoller;
+
+import com.team2357.frc2023.Robot;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class IntakeRollerEjectCubeCommand extends CommandBase {
+
```
Comment (NolanCampbell2357) [inline cowtown2023/src/main/java/com/team2357/frc2023/commands/intakeRoller/IntakeRollerEjectCubeCommand.java:8]: Not a big deal for these little commands, and now that we are not using singletons with long names, but keep in mind that you can set the intake subsystem to a local variable, that way you do not have to do Robot.intake everytime you want to reference it. Nothing needs to change here unless you want to.

File: cowtown2023/src/main/java/com/team2357/frc2023/subsystems/IntakeRollerSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    private CANSparkMax m_topIntakeMotor;
+    private CANSparkMax m_bottomIntakeMotor;
+
+    public IntakeRollerSubsystem() {
+        m_topIntakeMotor = new CANSparkMax(Constants.CAN_ID.TOP_INTAKE_ROLLER_MOTOR_ID, MotorType.kBrushless);
+        m_bottomIntakeMotor = new CANSparkMax(Constants.CAN_ID.BOTTOM_INTAKE_ROLLER_MOTOR_ID, MotorType.kBrushless);
+    }
+
+    public void configure() {
```
Comment (NolanCampbell2357) [inline cowtown2023/src/main/java/com/team2357/frc2023/subsystems/IntakeRollerSubsystem.java:20]: It does not look like you are calling this method. Also, I would be okay with getting rid of this method in our subsystems and moving the config to the constructor.

File: cowtown2023/src/main/java/com/team2357/frc2023/Robot.java [hunk 3]
```diff
@@ -37,7 +37,7 @@ public class Robot extends LoggedRobot {
   public static SwerveDriveSubsystem drive;
   public static ShooterSubsystem shooter;
   public static IntakeSlideSubsystem slide;
-  public static IntakeSubsystem intake;
+  public static IntakeRollerSubsystem intake;
```
Comment (NolanCampbell2357) [inline cowtown2023/src/main/java/com/team2357/frc2023/Robot.java:40]: Shouldn't this be `intakeRoller` instead of just `intake`?

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

File: cowtown2023/src/main/java/com/team2357/frc2023/subsystems/IntakeRollerSubsystem.java (hunk 2 above)
Comment (Tyson453) [reply cowtown2023/src/main/java/com/team2357/frc2023/subsystems/IntakeRollerSubsystem.java:26]: I'm gonna keep it as a separate method for organization purposes

**Pull request merged**